Shopify/cli
# Fix ink dependency with yarn stable

### WHY are these changes introduced?
Fixes https://github.com/Shopify/cli/issues/2906



### WHAT is this pull request doing?
- Duplicate `ink` and `react` dependencies declaration inside the `dependencies` section.
- Keep dependencies inside the `peerDependencies` sections.
- Solution based on this [post](https://github.com/yarnpkg/berry/issues/3710#issuecomment-963732582)

### How to test your changes?
1. `yarn create @shopify/app --template none`
2. `yarn generate extension --template subscription_ui --flavor react`
3. `yarn dev`

You can run the same test for a remix app and using the other package managers as well and everything should work ok


### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
